JS-Algorithms/challenges.js

```diff
@@ -96,12 +122,24 @@ Since 10! === 3628800 and you sum 3 + 6 + 2 + 8 + 8 + 0 + 0
 
 const digitSum = (n) => {
   // YOUR CODE HERE...
+  let x = 1;
+  for(let i = n; i >= 1; i--) {
+    x = x * i
+  }
+  const getLongNumber = BigInt(x);
+
+  const string = getLongNumber.toString();
+  const arrString = string.split('');
```

Comment:
My general rule of thumb is to break out a variable if something is needed to be used in multiple places. 

Don't be afraid of higher order functions together 😄

I think the addedbenefitt to this is you don't have to think of more variable names 😂 

Looks like everything after line 128 can be done in a single line